libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigationProvider.kt

```diff
@@ -33,6 +36,22 @@ object MapboxNavigationProvider {
         return mapboxNavigation!!
     }
 
+    @VisibleForTesting
```

Comment:
Why do you need this?

Reply:
This is needed in order to provide mocked `TelemetryWrapper` for unit tests.